scripts/create_and_deploy_private_version.sh

```diff
@@ -87,6 +87,12 @@ if [[ "${CHANNEL}" = "" || "${NETWORK}" = "" || "${CONFIGFILE}" = "" || "${TEMPL
     exit 1
 fi
 
+# Don't use environment variable for S3_RELEASE_BUCKET - default to algorand-internal for private deployments
+if [[ ! -z "${S3_RELEASE_BUCKET}" && -z "${BUCKET}" ]]; then
+    echo "Ignoring S3_RELEASE_BUCKET setting - defaulting to algorand-internal.  Use -b to override."
+fi
+S3_RELEASE_BUCKET="${BUCKET:-algorand-internal}"
```

Comment:
nit: Could you set the `algorand-internal` default at the top of the file when the rest are initialized to empty? I think it will let you simplify the code a bit:

```
if [[ ! -z "${S3_RELEASE_BUCKET}" ]]; then
    echo "Ignoring S3_RELEASE_BUCKET setting - defaulting to algorand-internal.  Use -b to override."
fi

# If we really need this it would always be.
export S3_RELEASE_BUCKET="${BUCKET}"
```

Then just use `-b "${BUCKET}"` at the end.

Reply:
Initializing prevents the ability to report this warning.

Reply:
Given the users of this script I don't think it's a major point, but with the current logic `S3_RELEASE_BUCKET` could be silently overwritten with the default value.